cms/templates/partials/catalog-card.html

```diff
@@ -1,98 +1,7 @@
-                        ${{ courseware_page.product.first_unexpired_run.current_price|floatformat:"0"|intcomma }}
-                    </li>
-                {% endif %}
-                {% if object_type == "program" %}
-                    <li>
-                        <strong>Number of Courses:</strong>
-                        {{ courseware_page.product.courses.count }}
-                    </li>
-                {% endif %}
-                {% if courseware_page.duration %}
-                    <li>
-                        <strong>Duration:</strong>
-                        {{ courseware_page.duration }}
-                    </li>
-                {% endif %}
-                {% if courseware_page.product.next_run_date %}
-                    <li>
-                        <strong>Next Start Date:</strong>
-                        {{ courseware_page.product.next_run_date|date:"F j, Y" }}
-                    </li>
-                {% endif %}
-            </ul>
-        </div>
-        <div class="offering-type">
-            <label>
-                {% if object_type == "course" %}
-                    Single
-                {% else %}
-                    Multi
-                {% endif %}
-                Course Online
-            </label>
-        </div>
-    </div>
-    <div class="bottom">
-      {% with page_id=courseware_page.pk|stringformat:"i" %}
-        {% with collapse_id="collapsible_"|add:tab|add:"_"|add:page_id collapse_input_id="collapse_"|add:tab|add:"_"|add:page_id %}
-        <input type="checkbox" class="toggle" id="{{ collapse_input_id }}" data-target="#{{ collapse_id }}" data-toggle="collapse" />
-        <label for="{{ collapse_input_id }}" class="lbl-toggle">
-            Details
-        </label>
-        <div id="{{ collapse_id }}" class="collapse">
-          <div class="details-body">
-            {% if courseware_page.catalog_details %}
-              <div class="description">{{ courseware_page.catalog_details|richtext }}</div>
-            {% endif %}
-            {% if object_type == "program" %}
-              <ul class="program-course-links">
-                {% for course_page in courseware_page.course_pages %}
-                  <li>
-                    <a href="{% pageurl course_page %}">
-                      {{ course_page.title }}
-                    </a>
-                  </li>
-                {% endfor %}
-                </ul>
-                <a href="{% pageurl courseware_page %}" class="link-button">
-                  View Program
-                </a>
-            {% endif %}
-            {% if object_type == "course" %}
-              <a href="{% pageurl courseware_page %}" class="link-button">
-                View Course
-              </a>
-            {% endif %}
-          </div>
-        </div>
-        {% endwith %}
-      {% endwith %}
+        {% include "partials/card_details_top.html" %}
     </div>
+    {% include "partials/card_details_bottom.html" with card_type="tab" %}
```

Comment:
Fyi @ahmed-arbisoft `card_type` is passed as argument.

Reply:
@tasawernawaz please refer to my previous comment.